docs_sphinx/advanced/scheduling.rst

```diff
@@ -70,3 +70,36 @@ Adapted from http://stackoverflow.com/questions/3160699/python-progress-bar
                 sys.stdout.write("\n")
 
     net.run(duration, report=ProgressBar(), report_period=1*second)
+
+**"Standalone Mode" Text based progress bar on console**
+
+This needs a "normal" Linux console, i.e. it might not work in an integrated
+console in an IDE.
+
+Adapted from https://stackoverflow.com/questions/14539867/how-to-display-a-progress-indicator-in-pure-c-c-cout-printf
+
+::
+
+    from brian2 import *
+    set_device('cpp_standalone', directory='STDP_standalone')
```

Comment:
Please remove the import line (implicit in most of the documentation), as well as the `directory` argument (which is specific to the STDP example.